# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1766783081 -3600
# Parent  406669caf25fbbe0c054305c2cb4eafa6f0689e5
Bug 1976738 - Let the background and foreground color customization of the message compose affect also plaintext compositions, take 2.

The original patch doesn't quite cut it, so let's fix it so it also works in dark mode.

diff --git a/mail/components/compose/content/MsgComposeCommands.js b/mail/components/compose/content/MsgComposeCommands.js
--- a/mail/components/compose/content/MsgComposeCommands.js
+++ b/mail/components/compose/content/MsgComposeCommands.js
@@ -10798,11 +10798,13 @@ function loadPlainTextStyle() {
     return;
   }
 
   const editor = GetCurrentEditor();
-  editor.document.documentElement.style.backgroundColor =
-    Services.prefs.getCharPref("msgcompose.background_color", "");
-  editor.document.documentElement.style.color = Services.prefs.getCharPref(
+  editor.document.body.style.backgroundColor = Services.prefs.getCharPref(
+    "msgcompose.background_color",
+    ""
+  );
+  editor.document.body.style.color = Services.prefs.getCharPref(
     "msgcompose.text_color",
     ""
   );
 }
